x/ccv/provider/keeper/keeper.go

```diff
@@ -442,68 +501,68 @@ func (k Keeper) DeleteUnbondingOpIndex(ctx sdk.Context, chainID string, vscID ui
 }
 
 // GetUnbondingOpsFromIndex gets the unbonding ops waiting for a given chainID and vscID
-func (k Keeper) GetUnbondingOpsFromIndex(ctx sdk.Context, chainID string, valsetUpdateID uint64) (entries []types.UnbondingOp, found bool) {
+func (k Keeper) GetUnbondingOpsFromIndex(ctx sdk.Context, chainID string, valsetUpdateID uint64) (entries []types.UnbondingOp) {
 	ids, found := k.GetUnbondingOpIndex(ctx, chainID, valsetUpdateID)
 	if !found {
-		return entries, false
+		return entries
 	}
 	for _, id := range ids {
 		entry, found := k.GetUnbondingOp(ctx, id)
 		if !found {
+			// An error here would indicate something is very wrong.
+			// Every UnbondingOpIndex is assumed to have the corresponding UnbondingOps set in store.
+			// This is done in AfterUnbondingInitiated and InitGenesis.
 			panic("did not find UnbondingOp according to index- index was probably not correctly updated")
 		}
 		entries = append(entries, entry)
 	}
 
-	return entries, true
+	return entries
 }
 
 // GetMaturedUnbondingOps returns the list of matured unbonding operation ids
-func (k Keeper) GetMaturedUnbondingOps(ctx sdk.Context) (ids []uint64, err error) {
+func (k Keeper) GetMaturedUnbondingOps(ctx sdk.Context) (ids []uint64) {
 	store := ctx.KVStore(k.storeKey)
 	bz := store.Get(types.MaturedUnbondingOpsKey())
 	if bz == nil {
-		return nil, nil
+		return nil
```

Comment:
Should we instead panic here? Otherwise it might make sense to add a comment explaining why we don't

Reply:
I added a comment.